### PR TITLE
Adds Typing Indicator

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request

Based on the Baystation12 typing indicator:

When a human is in the process of typing via the "say" or "me" windows, they will gain a little speech bubble adjacent to their sprite indicating they are in the process of typing.

## Why It's Good For The Game
Encourages more verbal interaction. Players are less likely to walk off or do something else if they know someone they can see is typing.

## Changelog
:cl:
add: Humans now display a chat bubble when typing, as an effort to move us closer to Baystation
/:cl:

### CODE REVIEW

**Comment by francinum** [comment]
I am advancing my agenda, one step at a time 😏 

But really, this thing is kinda only vaguely reliable and is probably going to result in some pretty shitty play for a few days.

**Comment by PowerfulBacon** [comment]
Who do me verbs have a typing indicator?

**Comment by Lagomorphica** [comment]
finally I know when to attack after asking somebody a question 😈

**Comment by Kapu1178** [comment]
> 
> 
> Who do me verbs have a typing indicator?

me's are just as important as normal speech, and we should be encouraging our players to use it more.

**Comment by PowerfulBacon** [comment]
I didn't even know there was a hotkey for *me, who doesn't just type them in the command line to avoid the dialogs 😅

**Comment by ivanmixo** [comment]
> finally I know when to attack after asking somebody a question 😈

this is why we can't have nice things

**Comment by BrumalLake** [comment]
andd here's the guy who said that making the icon a var for all mobs was stinky a couple of days ago

**Comment by Kapu1178** [comment]
> 
> 
> andd here's the guy who said that making the icon a var for all mobs was stinky a couple of days ago

not what i said at all. i said putting it in human/Initialize() was stinky, but it looks like I had no choice.

**Comment by Kapu1178** [comment]
known bug: Sticky overlays happen when you attempt to open a say dialog while you already have a say dialog open

File: code/modules/mob/typing_indicator.dm [hunk 1]
```diff
@@ -1,8 +1,9 @@
 /mob/proc/create_typing_indicator()
```
Comment (francinum) [inline code/modules/mob/typing_indicator.dm:3]: ...If this only works on human why is this not a stub function and an override defined on `mob/living/carbon/human`

Comment (Kapu1178) [reply code/modules/mob/typing_indicator.dm:3]: Because I’m filter worded. Will do when I get home

**Comment by qwertyquerty** [comment]
I don't like this. :key:

**Review by francinum — APPROVED** [review]

**Comment by Kapu1178** [comment]
pulled master to unfuck DME file

**Pull request merged**